File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/payments/PagoPasajeroDto.java
package ps.com.viajeros.dtos.payments;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class PagoPasajeroDto {
    private Long idPayment;
    private Long idViaje;
    private String nombrePasajero;
    private BigDecimal monto;
    private String estadoPagoPasajero;
    private String nombreChofer;
    private String cbuChofer;
    private String bancoChofer;
    private String estadoPagoChofer;
}

File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Admin/PagoPasajeroDto.ts
export interface PagoPasajeroDto {
  idPayment: number,
  idViaje: number;
  nombrePasajero: string;
  monto: number;
  estadoPagoPasajero: string;
  nombreChofer: string;
  cbuChofer: string;
  bancoChofer: string;
  estadoPagoChofer: string;
}
